doc/api.md

```diff
@@ -26,19 +26,37 @@ kernelspec:
     Section of dsn_file to be used for generating a connection string
 
 ``-p`` / ``--persist``
-    Create a table name in the database from the named DataFrame
-
-``-n`` / ``--no-index``
-    Do not persist data frame's index
+    Create a table name in the database from the named DataFrame ([example](#create-table))
 
 ``--append``
-    Like ``--persist``, but appends to the table if it already exists 
+    Like ``--persist``, but appends to the table if it already exists ([example](#append-to-table))
 
 ``-a`` / ``--connection_arguments <"{connection arguments}">``
     Specify dictionary of connection arguments to pass to SQL driver
 
 ``-f`` / ``--file <path>``
-    Run SQL from file at this path
+    Run SQL from file at this path ([example](#run-query-from-file))
+
+```{versionadded} 0.4.2
+```
+
+``-n`` / ``--no-index``
+    Do not persist data frame's index (used with `-p/--persist`) ([example](#create-table-without-dataframe-index))
+
+```{versionadded} 0.4.3
+```
+
+``-S`` / ``--save <name>``
```

Comment:
Is there a mechanism to load it as well?

Reply:
just saw the with, the keyword is not too intuitive, maybe load?

Reply:
Also what happens if you have 2 or more?

Reply:
this is just missing documentation, this PR didn't touch any of these options

Reply:
Ok, do we need to open new issues on the questions above?

Reply:
you can, but ensure you read the docs to understand why things were named in a certain way and how they work: https://jupysql.readthedocs.io/en/latest/compose.html

Reply:
I see it's already in via this example: `%%sql --with genres_fav --with tracks_with_info` so no need.